### ISSUE TITLE
[세션#2] 기업에서의 오픈소스

### ISSUE THREAD

**Issue by wonderlandpark** [tangungithub/GDM2020-QnA#4]
### 세션#2 패널리스트
* 권민재 님: @mingrammer
* 옥찬호 님: @utilForever
* 이선협 님: @kciter

 ### 아래에 자유롭게 질문을 입력해주세요!

현재 국내 기업들에서 이너소스가 많이 활성화되어있다고 생각하시나요? 앞으로 오픈소스 문화는 어떻게 변화될까요?

**Comment by kciter** [comment]
이너소스에 대해 잘 모르더라도 GitHub와 같은 플랫폼을 기업이 많이 사용하게 되면서 자연스럽게 이너소스가 많이 퍼져있다고 생각합니다. 특히 스타트업처럼 레거시가 적은 기업은 대부분 이너소스가 활성화되어있다고 생각합니다.
최근 몇 년간 큰 기업 주도하에 진행되는 오픈소스 프로젝트가 상당히 많이 늘었습니다. 여러 가지 목적이 있겠지만 그로 인해 오픈소스 활동이 더 활발해진 것은 분명합니다. 그런 관점에서 생각해보면 앞으로 더욱더 많은 기업이 자사의 소스를 공개하는 경우가 더 많이 생기지 않을까 싶습니다. 그러면 어느 순간 개발자에게 있어서 오픈소스 활동이 필수 아닌 필수가 되는 시기가 올지도 모르겠네요. :)

**Comment by son7211** [comment]
네, 저도 같은 의견입니다. GitHub 플랫폼을 사용하시는 엔터프라이즈에서 협력하는 문화를 바꾸기 위해 많이 노력하고 있고, 많은 부분 바뀌었다고 생각합니다. 규모가 큰 조직에서는 팀단위, 프로젝트 단위등에서 변화를 위해 많은 노력을 기울이고 계시고 작은 조직도 이러한 이너소스 협업의 방향으로 노력을 많이 기울이시는 것으로 알고 있습니다~.
